resources/zib/zib-MedicalDevice.xml

```diff
@@ -275,6 +280,23 @@
         <comment value="Reversed reference for zib Stoma.StomaMaterial" />
       </mapping>
     </element>
+    <element id="DeviceUseStatement.reasonReference:wound">
+      <path value="DeviceUseStatement.reasonReference" />
+      <sliceName value="wound" />
+      <short value="Wound" />
+      <comment value="Please note that on a functional level, zib Wound references zib MedicalDevice, but in FHIR this direction is reversed." />
+      <alias value="Wond" />
+      <max value="1" />

```

Comment:
Remove this max cardinality, similar to the other slices on `.reasonReference`.

Reply:
Fix